doc/classes/PhysicsDirectBodyState.xml

```diff
@@ -57,7 +57,7 @@
 			<argument index="1" name="j" type="Vector3">
 			</argument>
 			<description>
-				Apply a positioned impulse (which will be affected by the body mass and shape). This is the equivalent of hitting a billiard ball with a cue: a force that is applied once, and only once. Both the impulse and the position are in global coordinates, and the position is relative to the object's origin.
+				Applies a positioned impulse to the body. An impulse is time independant! Applying an impulse every frame would result in a framerate dependant force. Thats why it should only be used, when simulating one-time impacts. The position uses the global coordinate system (the global rotation) but it is relative to the object's origin.
```

Comment:
"independent" and "dependent" are spelled wrong.
 > Thats why it should only be used, when simulating one-time impacts

"Thats (sic) why" -> "For this reason,"
Remove the comma after "used"